spec/features/signup_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature 'Signup Page Loads' do
```

Comment:
There's a mismatch between the feature description and what's being tested here I think?